TAO/tao/idl_features.h

```diff
@@ -57,13 +61,17 @@
 #endif
 
 #ifndef TAO_IDL_HAS_OCTET_AND_WCHAR_UNION_DISCS
-#  define TAO_IDL_HAS_OCTET_AND_WCHAR_UNION_DISCS 0
+#  define TAO_IDL_HAS_OCTET_AND_WCHAR_UNION_DISCS TAO_IDL_IDL_VERSION >= 0x40000
 #endif
 
 #ifndef TAO_IDL_HAS_STRUCT_INHERITANCE
 #  define TAO_IDL_HAS_STRUCT_INHERITANCE 0
 #endif
 
+#ifndef TAO_IDL_HAS_EMPTY_STRUCTS
+#  define TAO_IDL_HAS_EMPTY_STRUCTS TAO_IDL_IDL_VERSION >= 0x40000
```

Comment:
Shouldn't this be > 0x40000, as TAO4.0.0 has already been released?

Reply:
This is the IDL version, not TAO.

Reply:
Ok, missed that